### PR TITLE
Update the policy history to match the new status message format

### PR DESCRIPTION
**Pull request by mprahl**
Relates:
https://issues.redhat.com/browse/ACM-5175

### CODE REVIEW

**Comment by mprahl** [comment]
@dhaiducek and @yiraeChristineKim , this is required to make this PR pass https://github.com/stolostron/config-policy-controller/pull/524 but it also depends on code in this upstream PR https://github.com/open-cluster-management-io/config-policy-controller/pull/147.

File: test/e2e/configuration_policy_test.go [hunk 1]
```diff
@@ -214,10 +201,7 @@ var _ = Describe("Test configuration policy inform", Ordered, func() {
 			Expect(err).ToNot(HaveOccurred())
 			common.DoRootComplianceTest(rolePolicyName, policiesv1.Compliant)
 			expectedStatusMsgs = append(
-				[]string{
-					"Compliant; notification - roles [role-policy-e2e] in namespace default missing " +
-						"as expected, therefore this Object template is compliant",
-				},
+				[]string{"Compliant; notification - roles [role-policy-e2e] missing as expected in namespace default"},
 				expectedStatusMsgs...,
```
Comment (yiraeChristineKim) [inline test/e2e/configuration_policy_test.go:205]: is this `mustnothave` just double check

**Review by yiraeChristineKim — APPROVED** [review]

File: test/e2e/configuration_policy_test.go (hunk 1 above)
Comment (mprahl) [reply test/e2e/configuration_policy_test.go:205]: That's right. It is inform and `mustnothave`.

**Pull request merged**